content/millennium/r4/encounters/encounter.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Encounter | R4 API
+---
+
+# Encounter
+
+* TOC
+{:toc}
+
+## Overview
+
+The Encounter resource provides admissions or visits during which health care services were provided to a patient. An encounter has a class to distinguish between different health care settings such as inpatient, outpatient, emergency, etc. A patient may have one medical record number with multiple encounter numbers per facility or organization. There is substantial variance between organizations in the definition of an encounter and what events are aggregated together to constitute an encounter.
+
+## Patch
+
+Patch an existing encounter.
+
+    PATCH /Encounter/:id
+
+_Implementation Notes_
+
+* Only operations on the paths listed below are supported.
```

Comment:
Add an implementation note about this being a JSON Patch implementation.
`* This implementation follows the [JSON PATCH](https://tools.ietf.org/html/rfc6902) spec.`

Reply:
Added. https://github.com/cerner/fhir.cerner.com/pull/240/commits/2f088fe9f198e6cb9d48c1be55d513f5eb8c9f72#diff-6590262e197180cc09f6ad70a2a01a5bR22